client/src/containers/interventions/new/step1/component.tsx

```diff
@@ -107,6 +109,8 @@ const Step1: FC = () => {
     dispatch(resetInterventionData());
   }, [dispatch]);
 
+  console.log(watch());
```

Comment:
do not forget to remove console.log

Reply:
FYI ;) https://eslint.org/docs/rules/no-console